cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -310,6 +310,12 @@ spec:
         # Set the path where the Rook agent can find the flex volumes
         # - name: FLEXVOLUME_DIR_PATH
         #  value: "<PathToFlexVolumes>"
+        # Set the path where kernel modules can be found
+        # - name: LIB_MODULES_DIR_PATH
+        #  value: "<PathToLibModules>"
+        # Mount any extra directories into the container
+        # - name: EXTRA_MOUNTS
```

Comment:
What if we name the env var `AGENT_MOUNTS`? This way it is more obvious that it only applies to the agent.

Reply:
Yeah, sounds better. Done.